snakemake/config.json

```diff
@@ -1,9 +1,43 @@
 {
-  "output_dir": "data",
-
+  "output_dir": "../data",
+  "pams": {
+    "cas9": "NGG",
+    "cpf1": "TTTN"
```

Comment:
Let's make an issue on whether this should be `TTTN` or `TTTV` and clarify with Yuri/Henri.

Reply:
Ok - looks like we need to stick with `TTTN`. Closing this.